[2022-12-04 16:41:49,490] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-01-18T00:00:00+00:00 [queued]>
[2022-12-04 16:41:49,550] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-01-18T00:00:00+00:00 [queued]>
[2022-12-04 16:41:49,551] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 16:41:49,553] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-04 16:41:49,554] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 16:41:49,622] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): world> on 2022-01-18 00:00:00+00:00
[2022-12-04 16:41:49,644] {standard_task_runner.py:52} INFO - Started process 861 to run task
[2022-12-04 16:41:49,714] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'scheduled__2022-01-18T00:00:00+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpbv65naq2', '--error-file', '/tmp/tmpd850p_vc']
[2022-12-04 16:41:49,743] {standard_task_runner.py:80} INFO - Job 143: Subtask world
[2022-12-04 16:41:50,071] {task_command.py:369} INFO - Running <TaskInstance: first_dag.world scheduled__2022-01-18T00:00:00+00:00 [running]> on host fb90b48d1471
[2022-12-04 16:41:50,769] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-01-18T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-18T00:00:00+00:00
[2022-12-04 16:41:50,771] {logging_mixin.py:115} INFO - Airflow
[2022-12-04 16:41:50,772] {python.py:173} INFO - Done. Returned value was: None
[2022-12-04 16:41:50,810] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20220118T000000, start_date=20221204T164149, end_date=20221204T164150
[2022-12-04 16:41:50,879] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-04 16:41:50,980] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
